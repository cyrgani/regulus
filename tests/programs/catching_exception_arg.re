def(f, x, try_except(
    print(int(x)),
    print("error"),
)),

# should print to stdout
f(2),
f("a"),

# should not print to stdout
f(/(0, 0)),