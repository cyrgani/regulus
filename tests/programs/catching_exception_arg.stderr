
DivideByZeroError: attempted to divide by zero
at <stl:operators>:55:26
at <stl:operators>:53:11
at <stl:operators>:50:19
at programs/catching_exception_arg.re:11:4
at programs/catching_exception_arg.re:11:2